tests/stub/iteration/test_iteration_session_run.py

```diff
@@ -117,3 +117,66 @@ def test(version_, script_):
             for mode in ("write", "read"):
                 with self.subTest(version=version, mode=mode):
                     test(version, script)
+
+    @driver_feature(types.Feature.BOLT_4_4)
+    def test_nested(self):
+        uri = "bolt://%s" % self._server.address
+        driver = Driver(self._backend, uri,
+                        types.AuthorizationToken("basic", principal="",
+                                                 credentials=""))
+        self._server.start(path=self.script_path("v4x4",
+                                                 "pull_1_nested.script"))
+        session = driver.session("w", fetch_size=1)
+        res1 = session.run("CYPHER")
+        seq = []
+        seqs = []
+        i = 0
+        while True:
+            rec1 = res1.next()
+            if isinstance(rec1, types.NullRecord):
+                break
+            seq.append(rec1.values[0].value)
+            seq2 = []
+            res2 = session.run("CYPHER NESTED %d" % i)
+            while True:
+                rec2 = res2.next()
+                if isinstance(rec2, types.NullRecord):
+                    break
+                seq2.append(rec2.values[0].value)
```

Comment:
```suggestion
            res2 = session.run("CYPHER NESTED %d" % i)
            seq2 = [rec.values[0].value for rec in res2]
```

I'm fully aware that you just copy pasted this code that I just copy pasted while migrating tests from Python to TestKit :upside_down_face: But still. Also, feel free to do this change to `tests.stub.iteration.test_iteration_tx_run.TestIterationTxRun.test_nested`.